packages/rollup-plugin/MIT-LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2017 Honeybadger Industries LLC
+Copyright (c) 2016 Brandon Doran
```

Comment:
Is this necessary?

Reply:
I think so, but @joshuap is probably the best one to answer 🙂

Reply:
I copied the file directly from another package, so yeah I definitely defer to others on this!

Reply:
Yeah, if we are using other people's code, then we need to include them in the copyright notice.

Reply:
_and_ it needs to have had the same (MIT) license originally :)